pkg/model/logstore/logstore.go

```diff
@@ -371,6 +371,16 @@ func (s *LogStore) String() string {
 	return s.ManifestLog("")
 }
 
+func (s *LogStore) spansInManifest(mn model.ManifestName) map[SpanID]*Span {
```

Comment:
This wording is confusing to me. I'd maybe rename it to `spansForManifest`?

Reply:
done!